css/styles.css

```diff
@@ -6,13 +6,16 @@
 @import "/component/alert.css";
 @import "/component/badge.css";
 @import "/component/button.css";
+@import "/component/card.css";
 
 :root {
   --primary-color: #3066be;
   --primary-hover-color: #44b6eb;
   --secondary-color: #c5c3c6;
   --secondary-hover-color: #d2d1d3;
   --background-color: #ffff;
+  --secondary-background-color: #808080;
+  --third-color: #383838;
```

Comment:
If you are using `primary`, `secondary` then use `tertiary` as your third color name. Also, it would be good if you declare all the secondary variables one after the other.

Reply:
Done the Changes